Makefile

```diff
@@ -142,10 +143,10 @@ PCHS := ast/ast.hpp
 OBJ := $(addprefix $(OBJDIR),$(OBJ))
 
 
-all: $(BIN)
+all: $(MRUSTC)
 
 clean:
-	$(RM) -r $(BIN) $(OBJ) bin/mrustc.a
+	$(RM) -r $(MRUSTC) $(OBJ) bin/mrustc.a
```

Comment:
Seems risky to delete the user-provided mrustc on clean.
Might want to keep `BIN` around and default `MRUSTC` to the same value.